server/index.js

```diff
@@ -37,14 +37,24 @@ app.get('/api/allreviews/review/:id', (req, res) => {
   });
 });
 
+app.delete('/api/allreviews/review/:id', (req, res) => {
+  db.deleteReview(req.params.id, (err, data) => {
+    if (err) {
+      console.log('delete:', err);

```

Comment:
Looks like some superfluous console commands are still around.

If you're going to keep them, I think it's best to at least comment them out.